ibm/service/project/resource_ibm_project.go

```diff
@@ -784,7 +802,53 @@ func resourceIbmProjectMapToProjectConfigDefinitionPrototypeDAConfigDefinitionPr
 		model.EnvironmentID = core.StringPtr(modelMap["environment_id"].(string))
 	}
 	if modelMap["authorizations"] != nil && len(modelMap["authorizations"].([]interface{})) > 0 {
-		AuthorizationsModel, err := resourceIbmProjectMapToProjectConfigAuth(modelMap["authorizations"].([]interface{})[0].(map[string]interface{}))
+		AuthorizationsModel, err := ResourceIbmProjectMapToProjectConfigAuth(modelMap["authorizations"].([]interface{})[0].(map[string]interface{}))
+		if err != nil {
+			return model, err
+		}
+		model.Authorizations = AuthorizationsModel
+	}
+	if modelMap["inputs"] != nil {
+		model.Inputs = modelMap["inputs"].(map[string]interface{})
+	}
+	if modelMap["settings"] != nil {
+		model.Settings = modelMap["settings"].(map[string]interface{})
+	}
+	return model, nil
+}
+
+func ResourceIbmProjectMapToProjectConfigDefinitionPrototypeStackConfigDefinitionProperties(modelMap map[string]interface{}) (*projectv1.ProjectConfigDefinitionPrototypeStackConfigDefinitionProperties, error) {
```

Comment:
Is this func called anywhere to set into schema?

Reply:
I have regenerated the code based on the latest API spec, now it has been removed, it was a dirty function, I have re-run all test cases, output updated above